src/include/distributed/subplan_execution.h

```diff
@@ -32,6 +32,7 @@ extern void ExecuteSubPlans(DistributedPlan *distributedPlan);
 typedef struct IntermediateResultsHashEntry
 {
 	char key[NAMEDATALEN];
+	bool writeLocalFile;
```

Comment:
update the comments before the struct definition to describe what this is used for.